rc/.eslintrc.json

```diff
@@ -22,14 +23,18 @@
         "no-cond-assign": 2,
         "no-console": 2,
         "no-constant-condition": 2,
+        "no-continue": 0,
```

Comment:
No problem. This all shook out from trying to solve several problems with similar solutions.